node_cli/realm/realmAuth.js

```diff
@@ -0,0 +1,39 @@
+const Realm = require("realm");
+const index = require("../index");
+const config = require("../config");
+
+const REALM_APP_ID = config.realmAppId;
+const appConfig = {
+  id: REALM_APP_ID,
+  timeout: 10000,
+};
+
+const app = new Realm.App(appConfig);
```

Comment:
It seems like we could get around the need for passing the `authedUser` around between views if we had direct access to `app.currentUser`. Can we export `app` here to call it from the views or perhaps create a new `getCurrentUser()` method?